core/controllers/reader.py

```diff
@@ -330,12 +330,19 @@ def post(self, unused_exploration_id):
         """
         # A domain object representing the old state.
         old_state = exp_domain.State.from_dict(self.payload.get('old_state'))
+        # The id of the exploration to which the state belongs.
+        exp_id = self.payload.get('exp_id')
+        # The version of the exploration to which the state belongs.
```

Comment:
specifically, the version of the exploration when the classifier model was created.